src/linagora.esn.contact/app/contact/create/contact-create.pug

```diff
@@ -1,7 +1,8 @@
-sub-header
-  contact-create-subheader
+.modal.full-screen(tabindex="-1", role="dialog")
```

Comment:
the template is not part of the issue

Reply:
yes , i know i just added it to test the behaviour of modal